apollo-router/src/configuration/mod.rs

```diff
@@ -246,6 +251,19 @@ pub(crate) enum QueryPlannerMode {
     Both,
 }
 
+/// Apollo usage report signature and referenced field generation modes.
+#[derive(Clone, PartialEq, Eq, Default, Derivative, Serialize, Deserialize, JsonSchema)]
+#[derivative(Debug)]
+#[serde(rename_all = "lowercase")]
+pub(crate) enum ApolloSignatureNormalizationAlgorithm {
```

Comment:
This was a consequence of the way we had to do the weird import of the module in the fuzz test:
```
#[path = "../../apollo-router/src/apollo_studio_interop/mod.rs"]
mod apollo_router_usage_reporting;
use apollo_router_usage_reporting::generate_usage_reporting;
use apollo_router_usage_reporting::SignatureNormalizationAlgorithm;
use apollo_router_usage_reporting::UsageReportingComparisonResult;
```

Because of that I was unable to use the enum defined in the configuration module in the interop module. I had a DM conversation with Jeremy and Simon and the separate enums was the best workaround we could think of.

There's another thread on why we needed to do the import that way somewhere and I try to can find it if you're interested but the end result was that we just had to do it that way.